tooling/bundler/src/bundle/windows/sign.rs

```diff
@@ -132,10 +132,95 @@ pub fn verify(path: &Path) -> crate::Result<bool> {
   Ok(cmd.status()?.success())
 }
 
+fn parse_custom_sign_command(command: &str) -> crate::Result<Vec<String>> {
+  let mut result = Vec::<String>::new();
+
+  let mut buffer = String::new();
+
+  #[derive(Copy, Clone)]
+  enum State {
+    Idle,
+    // Parsing unquoted element
+    Unquoted,
+    // Parsing quoted element
+    Quoted,
+    // just after '"' parsing quoted element
+    // expecting '"' or whitespace and error otherwise
+    // Example: "foo"bar is error
+    // We accept '""' as a way to escape a quote
+    QuotedJustAfterQuote,
+  }
+
+  let mut state = State::Idle;
+
+  for c in command.chars() {
+    match (state, c) {
+      (State::Idle, c) if c.is_ascii_whitespace() => {
+        // skip whitespace
+      }
+      (State::Idle, '"') => {
+        state = State::Quoted;
+      }
+      (State::Idle, c) => {
+        buffer.push(c);
+        state = State::Unquoted;
+      }
+
+      (State::Unquoted, c) if c.is_ascii_whitespace() => {
+        result.push(buffer.clone());
+        buffer.clear();
+        state = State::Idle;
+      }
+      (State::Unquoted, '"') => {
+        return Err(anyhow::anyhow!("custom signing command contains an unclosed quote").into());
```

Comment:
We shouldn't error here imo, and just continue parsing

Reply:
I think it's a little unclear that `foo"bar baz` should be treated as `foo` `bar baz`, `foo"bar` `baz`, or `foo"bar baz` so I made this error. Which should I choose?

Reply:
it should be `foo"bar` `baz`

Reply:
but if we go with the object solution (which we should), this doesn't matter

Reply:
I mean your decision is not clear and reasonable enough I think.
For example, with command prompt (`cmd.exe`), `foo"bar baz` become `foobar baz`.


(Sorry I forget to list this up in the previous example list. `foo"bar baz` is mistake of `foobar baz `)


Anyway I made PR for object solution so this discussion might be useless.

#10634

Reply:
thanks, will close this PR then and move discussion there